src/pytest_html/scripts/utils.js

```diff
@@ -0,0 +1,18 @@
+const { manager } = require('./datamanager.js')
+
+const dayjs = require('dayjs')
+const duration = require('dayjs/plugin/duration')
+dayjs.extend(duration)
+
+const formatDuration = (dur) => {
```

Comment:
Försöker förstå vilket syftet med dayjs.duration är.
Är det för att göra 6100ms till 1min 1sekund, istället för, säg 61 sekunder?

Om det är fallet så klarar vi oss bra med native Date, enl: 

```
const dateobj = new Date(durationInMilliseconds);
const hours = dateobj.getHours() - 1
const elaspsedTime =  `${hours ? `H: ${hours}`: ''` M: ${date.getMinutes()} S: ${date.getSeconds()}`
```

Reply:
Nä, det är för att man ska kunna spec:a sitt eget date format: https://pytest-html.readthedocs.io/en/latest/user_guide.html#formatting-the-duration-column